File: module_simple/src/main/java/org/live/simple/MainActivity.java
package org.live.simple;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by wl on 2018/11/26.
 */
public class MainActivity extends AppCompatActivity {

    @BindView(R.id.tv_content)
    TextView tvContent;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
        tvContent.setText("new text");

        Intent intent = new Intent();
        intent.setClass(this, GreenDaoActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/org/live/nonokt/MainActivity.kt
package org.live.nonokt

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var intent: Intent = Intent()
        intent.setClass(this@MainActivity,TestActivity::class.java)
        startActivity(intent)
        finish()
    }
}